api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -198,8 +200,11 @@ public void setUp() {
     PROVIDED_CLOCK.setInstant(START_INSTANT);
   }
 
-  @Test
-  public void test_updateUserWithRetries_never_registered() {
+  @ParameterizedTest
```

Comment:
Took existing service tests and parameterized them to show the test when flag is both false and true.